content/tracking/tracking_1.md

```diff
@@ -165,6 +165,60 @@ export class SnowplowService {
   })
 }
 ```
+{{% /tab %}}
+{{% tab name="GTM" %}}
+❗❗**NOTE: A Google Tag Manager Account is required for this pathway**
```

Comment:
Can we use the notice shortcodes instead of these double exclamation marks across this accelerator? 

https://learn.netlify.app/en/shortcodes/notice/

We've got one on the streamlit page so I guess they will work everywhere else? If so, can we update all the pages with warning notes like this one to use the shortcodes?

Reply:
At the moment tabs don't support nested shortcodes which is why we settled on the double exclamation marks. Shall I spend some time trying to make this possible?

Reply:
Doesn't seem like a quick fix so I'll get this closed off first and take a look.